docs/recording/index.md

```diff
@@ -1,5 +1,11 @@
 # Recording with CARET
 
+## Introduction
+
+Recording includes only two steps; building an target application from source code and executing the target application. After execution, you will find trace data which has recorded events captured by trace points.
```

Comment:
It seems that this step does not include an explanation about recording.

1. building an target application
2. executing the target application
3. recording the target application

Reply:
@hsgwa  
Yes, 4 steps at maximum, but 2 steps at minimum. 
If CARET is applied to a smaller application, what an user has to do is building an target application and running it.

Reply:
If we are counting RECORD as one step, I feel it would be better to count it as 3 steps.

Reply:
@hsgwa  
I added the following sentences to meet your feedback.

```
Trace data validation is necessary as the third step because any failure on recording prevents users from analyzing performance correctly. Trace data is not recorded correctly due to any reason if a target application is large and complicated. CARET serves functions to validate trace data for the step.

CARET supports trace filter to exclude certain nodes or topics during recording. It is optional, but strongly recommended for analysis of a large application.
```

Reply:
It is good about what I pointed out.
I made an additional comment.